Lib/Inc/CUartCom.h

```diff
@@ -17,12 +17,15 @@
 #include "IComChannel.h"
 #include "usart.h"
 
-#define UART_TIMEOUT      100
-#define MAX_RX_QUEUE_SIZE 40
-#define MAX_TX_QUEUE_SIZE 40
-#define MAX_UART_ENGINES  8
-#define TX_BUF_SIZE       (MAX_STRING_SIZE + 1)
-#define RX_BUF_SIZE       60
+#define UART_TIMEOUT     100
+#define MAX_UART_ENGINES 8
+
+/**
+*@note TX_BUF_SIZE needs to be greater than MAX_STRING_SIZE
+*/
+#define TX_BUF_SIZE      (MAX_STRING_SIZE + 1)
```

Comment:
I recommend making Tx buffer about 500 to 1000 bytes, Rx buffer can be smaller since we are draining it very fast. Say 100-200.

Sometime in the future, we'll implement high water mark for buffers to see how they are utilised and then we'd be able to optimise the size.